widget.json

```diff
@@ -7,6 +7,7 @@
   "provider_only": false,
   "references": [],
   "html_tag": "div",
+  "support_article": "http://help.fliplet.com/fliplet-studio-2/how-to-configure-and-test-google-analytics"
```

Comment:
I was going to ask you to help me pull that variable into a standard part of the component UI as I realised hardcoding a support link in manually for each component is a poor way to standardise the integration of inline help.